### PR TITLE
Feature/tao 5828/sprint69/task status for compile delivery

### PR DESCRIPTION
**Pull request by tikhanovichA**
https://oat-sa.atlassian.net/browse/TAO-5828

### CODE REVIEW

File: scripts/update/Updater.php [hunk 1]
```diff
@@ -140,6 +140,36 @@ public function update($initialVersion)
             $this->setVersion('0.13.0');
         }
 
+
         $this->skip('0.13.0', '0.13.2');
+
+        $this->skip('0.13.2', '0.13.1');
```
Comment (gyszucs) [inline scripts/update/Updater.php]: This line is not needed.

File: scripts/update/Updater.php [hunk 2]
```diff
@@ -140,6 +140,36 @@ public function update($initialVersion)
...
+
+        if ($this->isVersion('0.13.1')) {
```
Comment (gyszucs) [inline scripts/update/Updater.php]: It should be 0.13.2

**Review by gyszucs — CHANGES_REQUESTED** [review]
Furthermore, could you please fix the unit tests as well? Some of them do not work. Btw, nice work, thanks. ;)

**Comment by tikhanovichA** [comment]
Fixed :)

**Review by gyszucs — APPROVED** [review]

**Pull request merged**